src/content/app/species-selector/components/species-search-field/SpeciesSearchField.tsx

```diff
@@ -75,16 +71,19 @@ If no results are shown, please try a different spelling or attribute
 // Species search field, but wrapped in redux.
```

Comment:
This comment is no longer correct (the search field is no longer using redux).

Rather, it should say, `Species search field, but wrapped in a component that reads a query parameter from the url`.

_(The reason this wrapped component exists is because in BLAST, the search field will behave differently)._